lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
+---
+_template:
+  nexus:
+    get_command: "show running monitor all"
+
+all_sessions:
+  multiple:
+  get_context: ~
+  get_value: '/^monitor session (\d+)$/'
+
+create:
+  set_value: 'monitor session %s'
+
+description:
+  kind: string
+  get_context: ["monitor session <id>"]
+  get_value: '/^description (.*)/'
+  set_context: ["monitor session <id>"]
+  set_value: '<state> description <desc>'
```

Comment:
The key here `<desc>` needs to change to `<description>` so that it matches your setter below.